sat/cli/bootprep/input/image.py

```diff
@@ -135,29 +170,27 @@ def configuration_group_names(self):
         return self.image_data.get('configuration_group_names')
 
     @property
-    def ims_data(self):
-        """dict: the data that defines the base IMS image or recipe"""
-        return self.image_data['ims']
-
-    @property
+    @abstractmethod
     def base_is_recipe(self):
         """bool: whether the starting point is a recipe in IMS or an image"""
-        return self.ims_data['is_recipe']
+        pass
```

Comment:
FWIW, the `pass` statements here and in the other abstract methods aren't necessary; the docstring is the entire function body.

Reply:
I guess it just looks off to me without that `pass`. I can remove it though